### ISSUE TITLE
user.config file parameter WindowTop at -25600

### ISSUE THREAD

**Issue by DuckEater** [omeryanar/FileExplorer#31]
It's been few days FileExplorer was no longer showing up. Version 2.0 and updating today to 2.1 made no difference.

I had 2 user.config files saved under : 
- "C:\Users\******\AppData\Local\FileExplorer\FileExplorer.exe_Url_mkes4jrtz0ikzi0mkcsuqhfyivv1nz54\1.0.0.0\user.config"
- "C:\Users\******\AppData\Local\FileExplorer\FileExplorer.exe_Url_1alekd13nw2gtnqezfj3rrdmsypqw4lb\1.0.0.0\user.config"

and deleting those user config files made FileExplorer work again.

Digging more into those files, I found something that looks just like a guilty parameter :
```
<setting name="WindowTop" serializeAs="String">
    <value>-25600</value>
</setting>
<setting name="WindowLeft" serializeAs="String">
    <value>-25600</value>
</setting>
```

I had previously a 4k monitor attached, but -25k seems a little bit extreme.

**Comment by DuckEater** [comment]
Digging more into those files, I found something that looks just like a guilty parameter :
```
<setting name="WindowTop" serializeAs="String">
    <value>-25600</value>
</setting>
<setting name="WindowLeft" serializeAs="String">
    <value>-25600</value>
</setting>
```

**Comment by omeryanar** [comment]
This is weird. FileExplorer saves the position and size of the current window in order to open a new window in the same size and location. In your log file, the LocationChanged event somehow returned a negative value. I've added an extra control that checks for top and left values greater than zero.

**Comment by omeryanar** [comment]
Fixed in the latest release (v2.2)
